events-sample/AndroidManifest.xml

```diff
@@ -0,0 +1,31 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="org.gearvrf.input.events"
+    android:versionCode="1"
+    android:versionName="1.0" >
+
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
@rahul27 READ_EXTERNAL_STORAGE is missing; per https://developer.oculus.com/documentation/publish/latest/concepts/publish-mobile-manifest/